test/unit/news_article_test.rb

```diff
@@ -88,19 +88,46 @@ class NewsArticleTest < ActiveSupport::TestCase
 
     assert article.can_be_related_to_policies?
   end
+
+  test "can associate news articles with worldwide organisations" do
+    news_article = create(:news_article)
+    assert news_article.can_be_associated_with_worldwide_organisations?
+    assert news_article.worldwide_organisations.create(name: "Zimbabwean Embassy")
+  end
+
+  test "should be invalid when associating a worldwide organisation to a non-World-news-story news article type" do
+    non_world_news_story_news_article_types = [
+      NewsArticleType::NewsStory,
+      NewsArticleType::PressRelease,
+      NewsArticleType::GovernmentResponse,
+    ]
+
+    non_world_news_story_news_article_types.each do |news_article_type|
+      news_article = build(:news_article, news_article_type: news_article_type)
+      news_article.worldwide_organisations.build(name: "Zimbabwean Embassy")
+      refute news_article.valid?
+      assert news_article.errors[:worldwide_organisations].include?("must be blank")
+    end
+  end
 end
 
 class WorldNewsStoryTypeNewsArticleTest < ActiveSupport::TestCase
+  test "#world_news_story? returns true" do
+    article = build(:news_article_world_news_story)
+    assert article.world_news_story?
+  end
+
   test "non-English primary locale should be valid" do
     news_article = build(:news_article_world_news_story)
     news_article.primary_locale = 'fr'
     assert news_article.valid?
   end
 
-  test "#world_news_story returns true" do
-    article = build(:news_article_world_news_story)
-
-    assert article.world_news_story?
+  test "should be invalid when not associating a worldwide organisation" do
```

Comment:
I think our general consensus (and possibly style guide) is to avoid 'should'. (I realise there is a lot of it about in Whitehall). "is invalid without a worldwide organisation" (or "it is"). Same in the couple of others below.